### DISCUSSION TITLE
App shows double pictures

### DISCUSSION THREAD

**Discussion by script-js** [immich-app/immich#8816] [Q&A]
For some reason, the Immich app shows two versions of every picture and video. One says it's on the server, and one says that it's not backed up. I'm on a Pixel 6 with Android 14. Has anyone else seen this before, and how do I fix it?

![Screenshot_20240415-111213.png](https://github.com/immich-app/immich/assets/134544514/fe4c3943-5405-401a-adca-e2c8cc36198d)

**Comment by Nordlicht-13** [comment]
Are those pictures in a external library?
If so, try to "Remove offline files"
